docs/HowToGuides/GettingStarted.md

```diff
@@ -247,6 +247,13 @@ Phew, that's a lot to digest! Now let's proceed to the actual build itself!
    containing the Swift compiler and standard library and clang/LLVM build artifacts.
    If the build fails, see [Troubleshooting build issues](#troubleshooting-build-issues).
 
+   > **Note:**
+   > `--release-debuginfo` means that although debug information will be produced, all targets will
+   > be compiled in release mode, meaning optimized code, which can affect your debugging experience.
+   > Consider using `--release-debuginfo --debug-swift` to have only the swift targets (that includes `swift-frontend`)
+   > built in debug mode.
+   > To know more about each flag being passed to build script run `utils/build-script --help`.
+
```

Comment:
We already discuss this in https://github.com/apple/swift/blob/main/docs/HowToGuides/GettingStarted.md#debugging-issues. I'd vote for merging that one into yours (not necessarily in this PR), since the build configuration is really something you ought to decide on before invoking the build.

Reply:
Just adjusted to reference that section, let me know what you think.

Reply:
LGTM.